app/services/notifications/new_follower/follow_data.rb

```diff
@@ -0,0 +1,16 @@
+require "dry-types"
+require "dry-struct"
+
+module Notifications
+  module NewFollower
+    module Types
+      include Dry::Types.module
+    end
+
+    class FollowData < Dry::Struct
```

Comment:
I used `dry-struct` to check the input passed to the service to prevent errors like #2041. This gem can also be useful in the app when we need to check structure.

Reply:
Very clean. Love this.